api/src/wfl/module/covid.clj

```diff
@@ -17,6 +19,57 @@
 (defn ^:private get-snapshots-from-workspace
   [workspace])
 
+(defn throw-unless-column-exists [dataset table column-name]
```

Comment:
No action needed but planting the seed that this and `throw-unless-table-exists` could be good utility methods, if we find ourselves solving similar problems elsewhere in the code.

Reply:
Move the docstring between the name and parameters.

Reply:
Done